### ISSUE TITLE
Unable to extract the the putty ProxyPassword regkey

### ISSUE THREAD

**Issue by zenmonk** [osquery/osquery#6405]
osqueryi "SELECT data, path FROM registry WHERE key LIKE 'HKEY_USERS\%\Software\SimonTatham\PuTTY\Sessions\%\ProxyPassword'"

### What version of osquery are you using?
4.1.2

### What steps did you take to reproduce the issue?
Run the above query against any endpoint

### What did you expect to see?
Display the proxy password

### What did you see instead?
Command must be executed from an existing directory that you have "write" access to.

**Comment by theopolis** [comment]
@zenmonk, can you use `osqueryi` to run other questions?

For example does the following work?

```
osqueryi -A os_version
```

**Comment by zenmonk** [comment]
Yes it does work fine in other queries. I noticed that, the error pops up when the registry key doesn't exist. However, the reg key exists in this scenario. Very strange

**Comment by mike-myers-tob** [comment]
It sounds like the error is related to permissions on the registry key, but it will be difficult to reproduce or debug that unless we know what those permissions are. 

1) What is the error output if you run osquery with `--verbose`? 
2) Are you running osquery in an Administrator shell?
3) Has that user logged in? If not, could the problem be that the `HKEY_USERS` registry hive for that user might not be mounted?

> Command must be executed from an existing directory that you have "write" access to.

Directory? Executed from? Do you mean the query only works when the user that invoked osquery has write access to the registry key in the `WHERE` clause?
